Without this fix /sys RTC interface can't be used to program a wakeup alarm,
for example:
	echo enabled > /sys/class/rtc/rtc0/device/power/wakeup
	sec=`date '+%s'`
	let "sec=$sec+30"
	echo $sec > /sys/class/rtc/rtc0/wakealarm
	poweroff

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>

Index: linux-3.4.37/drivers/rtc/rtc-stmp3xxx.c
===================================================================
--- linux-3.4.37.orig/drivers/rtc/rtc-stmp3xxx.c	2013-04-10 11:16:54.000000000 +0200
+++ linux-3.4.37/drivers/rtc/rtc-stmp3xxx.c	2013-04-10 12:16:21.000000000 +0200
@@ -217,6 +217,7 @@
 			STMP3XXX_RTC_CTRL_ALARM_IRQ_EN,
 			rtc_data->io + STMP3XXX_RTC_CTRL_CLR);
 
+	device_set_wakeup_capable(&pdev->dev, 1);
 	rtc_data->rtc = rtc_device_register(pdev->name, &pdev->dev,
 				&stmp3xxx_rtc_ops, THIS_MODULE);
 	if (IS_ERR(rtc_data->rtc)) {
